marshmallow_jsonapi/schema.py

```diff
@@ -76,7 +76,22 @@ class Meta:
         pass
 
     def __init__(self, *args, **kwargs):
+        self.include_data = kwargs.pop('include_data', ())
         super(Schema, self).__init__(*args, **kwargs)
+        for field_name, field in self.fields.items():
+            if field_name in self.include_data:
+                if not isinstance(field, BaseRelationship):
+                    raise ValueError('Can only include relationships. "%s" is a "%s"'
+                                     % (field_name, field.__class__.__name__))
+                if not hasattr(field, 'schema') or not field.schema:
+                    raise ValueError('A schema is required to serialize "%s"' % field_name)
+                field.include_data = True
+            else:
+                field.include_data = False
```

Comment:
I don't think so, because unfortunately fields are only instantiated once, while the schema can be instantiated multiple times with different `include_data` arguments, so we need to have a way to set `include_data` of fields to `False`. I'll change it to
```
elif isinstance(field, BaseRelationship):
                field.include_data = False
```
That should prevent the mentioned error.